src/resonate/shells/poller.py

```diff
@@ -27,9 +27,7 @@ def __init__(
 
     def _run(self) -> None:
         while True:
-            poll_url = (
-                f"{self._url}:8002/{self._scheduler.logic_group}/{self._scheduler.pid}"
-            )
+            poll_url = f"{self._url}:8002/{self._scheduler.group}/{self._scheduler.pid}"
```

Comment:
Just want to make sure we don't forget to make the port configurable, I'm guessing this PR is for renames and we can get to this in the next one?

Reply:
Yes! This is next in line, together with creating a top level `Resonate` class